packaging/deb/builddebs.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+MAINTAINER="Your Name <you@example.com>"
```

Comment:
People using some kind of automation/process to produce release binaries (like one published in foundationdb.org), this default may end up in their release builds if they don't notice this patch. Not sure if its ok, but maybe we should error out if this is not explicitly set? Alternatively, we should mention this in release notes?

Reply:
It can quickly get complicated on such details... But I think notes would be good 👍 
We could also do something like:
`read -p "Input Package maintainer eg.: Your Name <you@example.com> [or hit enter for ${USER}]" MAINTAINER`
`if [ -z "$MAINTAINER" ]; then MAINTAINER="${USER}"; fi`